internal/gitClient/git.go

```diff
@@ -33,12 +33,12 @@ func CloneRepoAndDetokenizeTemplate(githubOwner, repoName, folderName string, br
 	directory := fmt.Sprintf("%s/%s", config.K1FolderPath, folderName)
 	err := os.RemoveAll(directory)
 	if err != nil {
-		log.Println("Error removing dir(expected if dir not present):", err)
+		log.Error().Err(err).Msg("Error removing dir(expected if dir not present):")
 	}
 
 	err = CloneTemplateRepoWithFallBack(githubOwner, repoName, directory, branch, tag)
 	if err != nil {
-		log.Panicf("Error cloning repo with fallback: %s", err)
+		log.Error().Err(err).Msg("Error cloning repo with fallback")
 	}
 	if err != nil {
 		log.Printf("error cloning %s repository from github %s", folderName, err)
```

Comment:
Are we stop to panic-ing, is downstream logic checked?
> this change impact the behavior..  Why we have 2 checks for the same error, any special reason for it?

Reply:
my bad, it should still be panic for the time being. later we can return the error for the top caller. updated in the commit